File: tort-core/src/test/java/com/mzr/tort/testsample/domain/entity/Student.java
package com.mzr.tort.testsample.domain.entity;

import com.mzr.tort.core.domain.FinishableLongIdEntity;

import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.ManyToOne;
import javax.persistence.Version;

@Entity
public class Student extends FinishableLongIdEntity {

    private String name;

    private Form form;

    private Long version;

    public Student() {
//        System.out.println("some");
    }

    public Student(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @ManyToOne(fetch = FetchType.LAZY)
    public Form getForm() {
        return form;
    }

    public void setForm(Form form) {
        this.form = form;
    }

    @Version
    public Long getVersion() {
        return version;
    }

    public void setVersion(Long version) {
        this.version = version;
    }

}

File: sample-tort-app/src/main/kotlin/com/mzr/tort/sample/domain/entity/Student.kt
package com.mzr.tort.sample.domain.entity

import javax.persistence.Entity
import javax.persistence.FetchType
import javax.persistence.ManyToOne
import javax.persistence.Version

import com.mzr.tort.core.domain.FinishableLongIdEntity

@Entity
class Student : FinishableLongIdEntity {

    var name: String? = null

    @get:ManyToOne(fetch = FetchType.LAZY)
    var form: Form? = null

    @get:Version
    var version: Long? = null

    constructor() {
    }

    constructor(name: String) {
        this.name = name
    }

}